bundles/AdminBundle/Resources/public/js/pimcore/document/editables/link.js

```diff
@@ -147,11 +158,45 @@ pimcore.document.editables.link = Class.create(pimcore.document.editable, {
         this.window.close();
     },
 
+    checkValue: function (mark) {
+        var data = this.getValue();
+        var text = '';
+
+        if (this.required) {
+            if (this.required === "linkonly") {
+                if (this.data.path) {
+                    text = this.data.path;
+                }
+            } else {
+                if (this.data.text && this.data.path) {
+                    text = this.data.text + this.data.path;
+                }
+            }
+
+            this.validateRequiredValue(text, this.element, this, mark);
+        }
+
+        this.validateElement(text);
+    },
+
+    validateElement: function (value) {
+        if (!value && this.required) {
+            this.element.addCls('invalid-document-element');
```

Comment:
Is `invalid-document-element` really required? css class is not defined and no usage anywhere.

Reply:
Removing the whole block then.